UserInterface.adoc

```diff
@@ -1,44 +1,92 @@
-=== User interface conventions
+This section aims to introduce the user to Zyn's UI elements and explain their behaviour in detail.
+
+// TODO: Virtual keyboard panel
+
 === Panels
+
 When you run Zyn-Fusion you'll be presented with this window:
 
+// TODO: Color the elements here and use the colors as help.
 image::imgs/bank-read.png[]
 
+// TODO: Each panel a color
 As you can see it is divided into multiple panels, each with their own use.
 When navigating between different _views_, you'll consistently see the status
 panel on top of the window, the navigation panel on the left, and the bottom the
 virtual keyboard.
 In the middle of all of those is the content view which you can expect to change
 when navigating around Zyn-Fusion
 
-.Status Panel
+==== Status Panel
+
 The status panel is where you'll find a few global parameters, the file menu,
 automation-learn/fine-tuning mode selectors, the panic button, master VU meter,
-and the information tray.
+and the information tray. It is always situated at the top of Zyn's UI.
+We will be reading the status panel right-to-left, since you'll most often be using
+the righthand side.
 
 Panic button::
     When getting started you'll want to be familiar with the panic button as
     it will instantly silence the audio produced by Zyn, no matter what state
     the synth is in. It's a safety device in case the sounds go out of control.
+
 Master VU meter::
     This meter will display the current audio output levels. It can be useful for a few things:
     * It'll let you decide on choosing an optimal sound level for your patch.
     * It might be helpful to find out which instance of Zyn-Fusion is currently making sounds, if you're working with a few of them at once.
     * It'll help you make sure your MIDI keyboard is connected properly.
+
+VOLUME slider::
```

Comment:
although the UI does use shoutcase (mostly for typographic reasons), I'm not 100% sure if it's the best for the manual.

Reply:
Alright, I'll switch it back!